api_app/analyzers_manager/file_analyzers/mwdb_scan.py

```diff
@@ -58,7 +58,8 @@ def run(self):
         else:
             try:
                 file_info = mwdb.query_file(query)
-            except Exception:
+            except Exception as e:
+                logger.info(e, stack_info=True)
```

Comment:
`logger.error` is called in the `_handle_exception` method which will be called since we are raising `AnalyzerRunException` so this here is redundant.

Reply:
ok so I do not like that we raise a so generic Exception. I'll dive deeper to what was the case the previous developer wanted to catch exactly and manage it without a AnalyzerRunException